README.md

```diff
@@ -1,160 +1,718 @@
 <p align="center"><a href="https://circleci.com/gh/elastic/search-ui/tree/master"><img src="https://circleci.com/gh/elastic/search-ui/tree/master.svg?style=svg&circle-token=c637bc2af60035a1f4cb5367071999ced238be76" alt="CircleCI buidl"></a>
 <img src="https://img.shields.io/badge/version-beta-red.svg" alt="BETA" /></p>
 
-> Libraries for the fast development of modern, engaging search experiences.
+> Libraries for the fast development of modern, engaging search experiences. :tada:
 
 ## Contents
 
+- [About Search UI](#about-search-ui-rocket)
 - [Getting started](#getting-started-)
-- [Developing Search UI](#developing-search-ui)
-- [Running tests](#running-tests)
+- [Configuration](#configuration)
+- [Customization](#customization)
 - [FAQ](#faq-)
 - [Contribute](#contribute-)
 - [License](#license-)
 
 ---
 
-## Getting started 🐣
+## About Search UI :rocket:
+
+Search UI allows you to quickly implement search experiences without re-inventing the wheel.
+
+It supports **[React](https://reactjs.org)** and **works with any search API**.
 
-Search UI supports any search engine provider and JavaScript framework.
+### Features :+1:
 
-It is developed and maintained by [Elastic](https://elastic.co).
+- **You know, for search**: Maintained by [Elastic](https://elastic.co), the team behind Elasticsearch.
+- **Speedy Implementation** - Build search with a search box, results view, sorting, and more, with a few lines of code.
+- **Customizable** - Tune the Components, markup, styles, and behaviors to your liking.
+- **Smart URLs** - Searches, paging, filtering, and more, are captured in the URL for direct result linking.
+- **Headless** - Leverage our application logic, provide your own Components or views.
+- **JavaScript-y** - Can be used used with any JavaScript library, even vanilla JavaScript.
```

Comment:
Is there a better term that "JavaScript-y"? It IS JavaScript, the term "JavaScript-y" makes it sound like it's ... only sort-of javascript?

Reply:
If this is based on a feature I had listed, the original intent was just to illustrate that you can use the `search-ui` library independently of React, with any Framework.